New log file opened at Wed Jul 27 14:00:38 CST 2016 by 8640@USER-20160226NZ
Taking next connection to: 8550
New client attempting to connect on port: 8551-8550
  Address = /10.10.10.245:8550
This client on port 8551 has uuid 774777a9-f28b-4a29-9149-e48faa0c9150 configName activator humanReadableName Activator protocolVersion 3 featureTags Vector()
Connected Clients: activator-774777a9-f28b-4a29-9149-e48faa0c9150
Taking next connection to: 8550
New client attempting to connect on port: 8606-8550
  Address = /10.10.10.245:8550
Handshake exception on socket: 8606
sbt.impl.ipc.HandshakeException: Unable to perform handshake
	at sbt.impl.ipc.Peer.handshake(IPC.scala:46)
	at sbt.impl.ipc.MultiClientServer.<init>(IPC.scala:150)
	at sbt.server.SbtServerSocketHandler$$anon$1.run(SbtServerSocketHandler.scala:43)
Caused by: java.net.SocketException: Software caused connection abort: recv failed
	at java.net.SocketInputStream.socketRead0(Native Method)
	at java.net.SocketInputStream.socketRead(SocketInputStream.java:116)
	at java.net.SocketInputStream.read(SocketInputStream.java:170)
	at java.net.SocketInputStream.read(SocketInputStream.java:141)
	at java.io.BufferedInputStream.fill(BufferedInputStream.java:246)
	at java.io.BufferedInputStream.read(BufferedInputStream.java:265)
	at java.io.DataInputStream.readInt(DataInputStream.java:387)
	at sbt.impl.ipc.Peer.receive(IPC.scala:95)
	at sbt.impl.ipc.Peer.handshake(IPC.scala:49)
	... 2 more
Taking next connection to: 8550
New client attempting to connect on port: 8607-8550
  Address = /10.10.10.245:8550
This client on port 8607 has uuid a907759a-b922-4271-ba24-2d4aff7aed98 configName play-fork-run humanReadableName Play Fork Run protocolVersion 1 featureTags Vector()
Connected Clients: activator-774777a9-f28b-4a29-9149-e48faa0c9150, play-fork-run-a907759a-b922-4271-ba24-2d4aff7aed98
Taking next connection to: 8550
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 8550
Checking to see if clients are empty...
We have 1 non-daemon clients, continuing serving connections.
Taking next connection to: 8550
